### ISSUE TITLE
Discord Contributor Moderation Policy (Proposal)

### ISSUE THREAD

**Issue by winniendela** [atomone-hub/genesis#98]
### Moderation Policy

**Welcome to the AtomOne Discord Channel!** 

This Discord server is dedicated to fostering collaboration, discussion, and innovation for the AtomOne project, a minimal alternative fork of Gaia ("cosmoshub4") running alongside Gaia to prepare for all contingencies, and also to operate as a political party base in relation to Gaia. Whether you're into blockchain, are a developer, validator, or just curious about AtomOne, this is the perfect place to connect, share ideas, and contribute to the project's growth.

### Server guidelines & code of conduct

To ensure a productive and respectful environment for all contributors, we adhere to the following guidelines:

- **Respect and inclusivity:** Treat all members with respect. We welcome diverse viewpoints and encourage constructive discussions. Respect the privacy and boundaries of all community members.

**- No harassment:** Harassment in any form is not tolerated. This includes, but is not limited to, bullying and use of derogatory language.

- **Stay on topic:** Please keep discussions relevant to the channel topics. This helps in maintaining clarity and efficiency in our conversations.

- **No spamming:** Avoid spamming messages, links, or images. Share content that contributes to the discussions.

### Channels overview

**#general:** For general discussions about AtomOne and the Cosmos ecosystem.
**#development:** Dedicated space for technical discussions, development updates, and collaboration.
**#validators:** A place for validators to discuss and coordinate.
**#tokenomics:** Discussions about AtomOne’s tokenomics, staking, and governance.
**#announcements:** Official updates and announcements from the AtomOne team.
**#support:** Get help from the community on any AtomOne related queries.
**#faq:** find answers to frequently asked questions and basic information about AtomOne. Ideal for new comers!
**#off-topic:** For non-AtomOne related chatter. Keep it friendly and fun!

### Getting help
If you have questions or need assistance, feel free to ask in the #support channel, if you’re unable to find answers in the #faq channel. AtomOne moderators and community members are here to help.

### Participate and contribute
Your insights, questions, and feedback are invaluable. We encourage active participation and look forward to your contributions. Together, we can drive innovation and growth in the Cosmos ecosystem. 

### Make PRs on GitHub
Contributing to the [AtomOne GitHub](https://github.com/atomone-hub/genesis) is a great way to get involved! Whether it's reporting issues, proposing new features, or improving documentation, every contribution counts. 

- [Vision and Mission](https://github.com/atomone-hub/genesis#vision-and-missions)
- [Objectives](https://github.com/atomone-hub/genesis#objectives)
- [Plan](https://github.com/atomone-hub/genesis#plan)
- [FAQs](https://github.com/atomone-hub/genesis#faq)
- [ToDo ](https://github.com/atomone-hub/genesis#todo)
- [Glossary](https://github.com/atomone-hub/genesis#terms)

### Enjoy your stay
We're excited to have you as part of the AtomOne community. Let's collaborate to make the AtomOne Discord channel a thriving hub for learning, sharing, and contributing to the AtomOne project.

Welcome aboard!

**Comment by Ticojohnny** [comment]
Hey @winniendela, this a great basis to work off of. I think once we have some further comments from the community we can link this to a pull request and hopefully get the Discord launched after the holidays and see where it goes.

**Comment by Ticojohnny** [comment]
Taking a second look, I think having some detailed expectations for a moderator and what to do in troublesome situations would be helpful. Also probably a good idea to create a private channels for moderators to come to consensus on whether an individual should be permabanned. 

Adding something like this could help.

### **Expectations Of A Moderator:** 

**Do your best to help community members with answers about AtomOne related questions:**

- If members are looking for documentation please point them to the repo.

- If you think you know exactly which issue the conversation is about, try to make that available. 

- If they're looking for specific info or reporting disruptions about any other project, please direct them to the official channel of that project

**Be in the channel consistently within reason to ensure that scams are not popping up, make sure to delete them as soon as you can.**

- A big part of the role of a moderator is to ensure that community members stay safe, it is always good to have an eye on the channel in case someone joins to spread spams, scams or misinformation.

**Ensure that you are not sharing rumors, and make an effort to guide conversations back to constructive, productive and building-focused chats**

- Conversations should be relevant to AtomOne, or GovNo

- If conversation steers away from AtomOne or GovNo in a way that is not bringing relevant or benefit to the conversation, try to invite the community members in the conversation to come back to the main topic or to continue the conversation in an off-topic channel.

**Try to ensure that people show respect each other, even while sharing differing opinions:**

- People should feel comfortable sharing thoughts and opinions. Abusive, oppressive or bullying attitude should not be welcome in the channels, please try to de-escalate before turning to more administrative means (i.e, mutes, removals)

**Avoid engaging with people who are there to create chaos and present disrespectful, condescending or inappropriate attitudes, try to stay detached and provide warnings:**

- See more in the [Disciplinary Measures] sections
- If it becomes evident that individuals are purposefully promoting their token or product without engaging in a constructive discussion related to AtomOne, please take appropriate action by muting them

**Encourage users to protect their privacy:** 

- Alert people that giving details about their location, email addresses and so on could be not the best decision. Delete the message if they do so.

**Encourage Developers and Validators to join our Discord Server and to use appropriate channels**

- The general channels under Lounge should stay entry-level, try to direct any technical questions, or validator inquiries to the #support and #validator channels respectively so that conversations can stay relevant to their categories.

**Encourage users to do their own research.**

- A lot of the time people will come in hoping to find an absolute answer, but most of the time that isn't possible, especially with questions regarding their own finances.** 

- Try to avoid giving absolute answers unless you are absolutely certain that is true.

- Always refer members to existing articles or groups that will more likely have the information that is relevant to what they are looking for

**Avoid Direct Messaging (DM) anyone, even if they reach out first:** 

- Please attempt to resolve all the eventual issues/disruptions within the chat room, unless the information exchanged is confidential in which case the moderator should ask the user to initiate the direct message.

**Avoid sending tokens to community members**

- This will set a precedence that is not fun to continue, encourage the use of faucets

**Share the latest announcement and the most important news from AtomOne's Github, or other future social platforms to encourage healthy discussions.**

### **Disciplinary Measures**

Cases can usually be divided into 3 categories:

Scams/Spam: Those who post scams or intentionally spam the channel with promotional content regarding projects not belonging to the AtomOne ecosystem.

Inappropriate Behavior: Those who use vulgar language or behave aggressively or inappropriately towards other users.

Toxic Behavior: Those who make heavy personal attacks, racism, sexism or discriminatory, defamatory or any other toxic behavior

**Scams/Spam**

- Scams and spam can come in many different forms:
 
- Content that is extremely irrelevant to AtomOne
 
- Talking about shitcoins, memecoins, weird Ethereum or BSC contracts
 
- Users offering financial advice, "Fred made me $5,000 dollars,"

- "User support" that links to wallets that aren't what is used for AtomOne

We should have a bot instantiated at some point to remove the scam/spam immediately, however if the bot happens to miss it, please delete the message and ban the user who posted it.

**Inappropriate Behavior**

Inappropriate behavior can be identified as:

- Person who ignores help and claims to have answers we cannot provide or claims to have them in an inappropriate way.

- People who insist on carrying on attitudes that are vulgar, intimidating, disrespectful or that may in any way make the rest of the community uncomfortable.

- People who do not observe the topic of the chat by intentionally going on tangential and borderline abusive conspiracy related conversation.

In these cases, the suggested behavior is as follows:

Warn the person that he/she will continue to persist in their behavior, they will be removed from the community. Once you warn someone, please mention it in the moderator's only channel.

After two warnings, no matter the time passed from the first and the second one, this person will be muted for 1 day. Then you can inform the person in the channel, inviting him/her to DM you to find a way to resolve this in the best way possible.

If the person proves to be uncooperative, uses other accounts to continue with his conduct despite the imposed limitation or after the period of limitation starts again with the unsuitable behavior, it will be necessary to mute the person again.

Please report this situation in the AtomOne Moderator's Only channel with the screenshot of the message that triggered the second mute and we can have a vote on permanent bans.

**Toxic Behavior**

These cases can be identified as:

- People who carry out personal attacks on other members of the channel
 
- People who use toxic language with references to racism, sexism and any sort of discrimination.

In these cases, please:

Mute the person immediately and report it to the AtomOne Moderator's only channel and we can have a vote on permanent bans.

**Comment by winniendela** [comment]
Yeah I wasn't so sure how much to flesh it out! These are great additions, great job!

**Comment by jaekwon** [comment]
What should the channels be? Also https://discord.gg/atomone for passerbys to join before it's officially open.

**Comment by moul** [comment]
![IMG_1108](https://github.com/atomone-hub/genesis/assets/94029/2fc1f168-e163-4ceb-91d8-28c46dd73c6b)

What about:
- removing #tokenomics, #governance
- renaming #atom-one-constitution to #constitution
- adding a #atomone-hub

Goal: we just have the 3 main projects (#constitution, #govno, #atomone-hub) and 2 special channels #validators and #support.

**Comment by POP24** [comment]
Hello everyone,

Giving a thumbs up to Manfred’s suggestions. To further enhance clarity and streamline our channels, I propose:

- Consolidation of Discussion Channels: Let’s consider merging similar channels in Lounge and Discussion to centralize conversations.

- Creating a "Tech" Category: I suggest grouping technical channels under a dedicated Tech category, including #github-feed, #tech-announcements, #testnet-faucet, and #get-your-tech-role to encourage and guide technical contributions.

Although I did not vote on the governance proposal, I am keen to be fully involved in the future of our community.

**Comment by POP24** [comment]
Also Support for New Members: For better integration, I suggest a structured #welcome-guide channel including #constitution, #about-ATOMONE, #official-links, and #FAQ, making it easier for new users to find essential information.

**Comment by WaqarMMirza** [comment]
> What should the channels be? Also https://discord.gg/atomone for passerbys to join before it's officially open.

Is it still open? I wasn't able to join.

**Comment by Ticojohnny** [comment]
@WaqarMMirza yes it should be, I’ve seen people join this morning. You’ll need to pass the captcha bot for now 

https://discord.gg/atomone

**Comment by WaqarMMirza** [comment]
> @WaqarMMirza yes it should be, I’ve seen people join this morning. You’ll need to pass the captcha bot for now
> 
> https://discord.gg/atomone

Thankyou, I was able to join.
